Tests/Clappr_Tests/Classes/Plugins/Core/MediaControl/MediaControlTests.swift

```diff
@@ -76,35 +78,39 @@ class MediaControlTests: QuickSpec {
 
             /*
              describe("#gestureRecognizer") {
-                it("toggles the viewcontroller's swipe to back gesture to true") {
-                    let viewController = UIViewControllerMock()
-                    let options: Options = [kMediaControlPlugins: [Seekbar.self]]
-                    let core = Core(loader: Loader(), options: options)
-                    let mediaControl = MediaControl(context: core)
-                    viewController.view.addSubview(mediaControl)
-                    mediaControl.render()
-                    if let seekbar = mediaControl.plugins.first(where: { $0.pluginName == Seekbar.name }) as? Seekbar {
-                        seekbar.view.frame = CGRect(x: 0, y: 0, width: 100, height: 100)
-                    }
-                    
-                    _ = mediaControl.gestureRecognizer(UITapGestureRecognizer(), shouldReceive: UITouch())
-                    
-                    expect(viewController.navigationController?.interactivePopGestureRecognizer?.isEnabled).to(beFalse())
-                }
-            }
+             it("toggles the viewcontroller's swipe to back gesture to true") {
```

Comment:
Why we have this a commented test?

Reply:
the tests were already commented, the diff was generated for indentation reason. U probably want to know why do we already have it, dont u? well, we decided to split the work of move media control and it basics plugins to clappr, but when we moved the media control layout manager we broken a few different tests for different type of plugins and to avoid big PR we decided to delegate those tests adjusts to each responsable plugin.

Reply:
Done!